static/js/lightbox.js

```diff
@@ -66,27 +80,80 @@ export class PanZoomControl {
         });
     }
 
+    constrainImage(e) {
+        // Instead of using panzoom's built in bounds option which was buggy
+        // at the time of this writing, we act on pan/zoom events and move the
+        // image back in to view if it is moved beyond the image-preview container.
+
+        const {scale, x, y} = e.getTransform();
+        const image_width = $(".zoom-element > img")[0].clientWidth * scale;
+        const image_height = $(".zoom-element > img")[0].clientHeight * scale;
+        const zoom_element_width = $(".zoom-element")[0].clientWidth * scale;
+        const zoom_element_height = $(".zoom-element")[0].clientHeight * scale;
+        const max_translate_x = $(".image-preview")[0].clientWidth;
+        const max_translate_y = $(".image-preview")[0].clientHeight;
+
+        // Adjust this to control the number of pixels of the image
+        // that should always be visible.
+        const return_buffer = 50 * scale;
```

Comment:
I don't quite understand this comment, but perhaps we should extract this 50 as a constant at the top of the PanZoomControl definition?  Also I don't understand whether this is total pixels or pixels in the minimum direction.

Reply:
Maybe we don't need to extract a constant now that I kinda understand what this does, but a clearer comment about how the dimensions work would be helpful.

Reply:
I've re-worded this comment block to hopefully make it easier to understand.